tests/sentry/processing/backpressure/test_redis.py

```diff
@@ -1,13 +1,30 @@
+import pytest
 from django.test.utils import override_settings
 
 from sentry.processing.backpressure.memory import iter_cluster_memory_usage
 from sentry.processing.backpressure.monitor import Redis, load_service_definitions
+from sentry.testutils.helpers import override_options
 
 
-def test_returns_some_usage() -> None:
-    with override_settings(SENTRY_PROCESSING_SERVICES={"redis": {"redis": "default"}}):
-        services = load_service_definitions()
+@override_settings(SENTRY_PROCESSING_SERVICES={"redis": {"redis": "default"}})
+def test_rb_cluster_returns_some_usage() -> None:
+    services = load_service_definitions()
+    redis_service = services["redis"]
+    assert isinstance(redis_service, Redis)
+
+    usage = [usage for usage in iter_cluster_memory_usage(redis_service.cluster)]
+    assert len(usage) > 0
+    memory = usage[0]
+    assert memory.used > 0
+    assert memory.available > 0
+    assert 0.0 < memory.percentage < 1.0
+
 
+@pytest.mark.redis_cluster
+@override_settings(SENTRY_PROCESSING_SERVICES={"redis": {"redis": "default-cluster"}})
+@override_options({"redis.clusters": {"default-cluster": {"is_redis_cluster": True}}})
+def test_redis_cluster_cluster_returns_some_usage() -> None:
+    services = load_service_definitions()
```

Comment:
for this test -- do we expect the same `isinstance`? shouldn't it be RedisCluster ?

Reply:
This service returns `Redis` or `RabbitMq`